src/lib.rs

```diff
@@ -25,6 +25,181 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/*!
```

Comment:
Should we add a `#[deny(missing_docs)]`?

Reply:
I'd love to do that eventually but right now that would make the build fail until we added quite some documentation. I opened https://github.com/pingcap/raft-rs/issues/72 to track this.